etc/picongpu/titan-ornl/gpu_picongpu.profile.example

```diff
@@ -30,33 +30,33 @@ export FC=$(which ftn)
 #export LD="/sw/xk6/altd/bin/ld"
 
 # symbol bug work around (should not be required)
-#MY_CRAY_LIBS=/opt/gcc/4.9.3/snos/lib64
+#MY_CRAY_LIBS=/opt/gcc/5.3.0/snos/lib64
 #export LD_PRELOAD=$MY_CRAY_LIBS/libstdc++.so.6:$LD_PRELOAD
 #export LD_PRELOAD=$MY_CRAY_LIBS/libgomp.so.1:$LD_PRELOAD
 #export LD_PRELOAD=$MY_CRAY_LIBS/libgfortran.so.3:$LD_PRELOAD
 
 # required tools and libs
 module load git
-module load cmake3/3.10.0  # missing
-module load cudatoolkit  # 7.5.18
+module load cmake3/3.11.3
+module load cudatoolkit  # 9.1.85
 # might fail to link with missing symbols:
-#   C++11 module rebuild pending [CCS #366279]
-module load boost/1.62.0
+#   C++11 module rebuild pending [CCS #389072]
```

Comment:
note: better compile your own boost for now, see instructions here:
https://github.com/ComputationalRadiationPhysics/picongpu/blob/b2188482c7f8b325901ad8ce4718b362772bf1e9/INSTALL.rst#boost